src/main/java6/net/finmath/marketdata/model/curves/ForwardCurveInterface.java

```diff
@@ -15,16 +15,23 @@
 public interface ForwardCurveInterface extends CurveInterface {
 
 	/**
+	 * Returns the forward for the corresponding fixing time without a given model. Note that this does not work for all interpolation methods as some of them require a model
+	 * @param fixingTime The fixing time of the index.
+	 * @return The forward
+	 */
+	double getForward(double fixingTime);
```

Comment:
This is my biggest concern (maybe my only one): I would like to enforce that we always have the "model" (context). This would allow to use curves "mapping" to others by name - which is a good feature (also used in calibration, where some curves change, but others not).